vcpkg.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
+  "name": "chatterino",
+  "version": "2.0.0",
+  "dependencies": [
+    "benchmark",
+    "boost-foreach",
+    "boost-interprocess",
+    "boost-random",
+    "boost-variant",
+    "gtest",
+    "openssl",
```

Comment:
openssl defaults to 3.0.2 from vcpkg, had to lock it to 1.1.1 while testing to produce working build
```
  "overrides": [
        { "name": "openssl", "version-string": "1.1.1n" }
    ]
```
other than that works fine and would put it as preferred build way for windows 👍

Reply:
Could you share some logs? I had no issues with the default version.

Reply:
it did build, but failed on all network https connections on startup. Figured it might be openssl since conan is also locked to 1.1.1, didn't troubleshoot further after it worked.

Reply:
No need, it somehow used the openssl outside of vcpkg (which was 1.1.1). I updated the dependencies.